lib/src/widgets/release_body.dart

```diff
@@ -0,0 +1,124 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_gen/gen_l10n/app_localizations.dart';
+import 'package:fwupd/fwupd.dart';
+import 'package:provider/provider.dart';
+import 'package:yaru_icons/yaru_icons.dart';
+
+import '../../device_model.dart';
+import '../../fwupd_x.dart';
+import 'confirmation_dialog.dart';
+import 'release_card.dart';
+
+class ReleaseBody extends StatelessWidget {
```

Comment:
naming is hard. :) what do you think about calling it `ReleasePage`? this feels more like an entire "page" than the "body" of something that could be combined with a "header" or "footer".

Reply:
The same is probably true for `DeviceBody` - I'll promote them to pages. They probably shouldn't be in the `widgets` directory anymore either.

Reply:
true, `src/widgets` was meant for standalone widgets that wouldn't depend on view models